### ISSUE TITLE
Navigations saving

### ISSUE THREAD

**Issue by supun-io** [hyvor/blogs#96]
TLDR: Editing footer navigation (Settings -> Navigation) saves it as a header item (probably the initial state is not taken into account in the component)

Report: "When changing the contact footer navigation's settings, it automatically changes the "Type = Header" even though we are editing a footer item. If that makes sense? So, each time we edit the contact footer, we need to re-mark it as a footer item or else it'll be auto-converted to a header item. I've reproduced this with any footer item I create; any time it's edited, it turns into a header item. But, we can stop it if, before finishing the edit, we click type and select "Footer" before clicking Update."
